<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDIT</class>
 <widget class="QWidget" name="EDIT">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EDIT</string>
  </property>
  <widget class="QPushButton" name="addB">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateB">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteB">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Path</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="reloadB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>RELOAD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
